lib/ansible/plugins/connection/network_cli.py

```diff
@@ -445,7 +452,8 @@ def send(self, command, prompt=None, answer=None, newline=True, sendonly=False,
             return to_text(response, errors='surrogate_or_strict')
         except (socket.timeout, AttributeError):
             display.vvvv(traceback.format_exc(), host=self._play_context.remote_addr)
-            raise AnsibleConnectionFailure("timeout trying to send command: %s" % command.strip())
+            raise AnsibleConnectionFailure("timeout: value is %s seconds trying to send command: %s"
```

Comment:
The error message can be "timeout value %s seconds reached while trying to send command %s"